### PR TITLE
Client login page

### PR DESCRIPTION
**Pull request by ReissJarvis**
Created the login page that allowed the user to input sessionId and call off to a service.

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
+import { SessionService } from './session.service';
+import { HTTP_PROVIDERS } from '@angular/http';
 
 @Component({
     moduleId: module.id,
     selector: 'app-root',
     templateUrl: 'app.component.html',
-    styleUrls: ['app.component.css']
+    styleUrls: ['app.component.css'],
+    providers: [HTTP_PROVIDERS, SessionService]
```
Comment (tsangste) [inline src/app/app.component.ts]: move the providers into the module

File: src/app/client-arrival-view/client-arrival-view.component.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+@Component({
+  moduleId: module.id,
+  selector: 'app-client-arrival-view',
+  templateUrl: 'client-arrival-view.component.html',
+  styleUrls: ['client-arrival-view.component.css']
+})
+export class ClientArrivalViewComponent implements OnInit {
+
+    @Input() name: string;
```
Comment (tsangste) [inline src/app/client-arrival-view/client-arrival-view.component.ts]: some of the tabbing looks wrong in this file

File: src/app/session.service.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Http } from '@angular/http';
+
+@Injectable()
+export class SessionService {
+
+    constructor(private http: Http) { }
```
Comment (tsangste) [inline src/app/session.service.ts]: no space in {}

File: src/app/waitingroom/waitingroom.component.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+@Component({
+  moduleId: module.id,
+  selector: 'app-waitingroom',
+  templateUrl: 'waitingroom.component.html',
+  styleUrls: ['waitingroom.component.css']
+})
+export class WaitingRoomComponent implements OnInit {
+
+  constructor(private router: Router) { }
```
Comment (tsangste) [inline src/app/waitingroom/waitingroom.component.ts]: tabbing looks wrong here

File: src/app/waitingroom/waitingroom.component.ts [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit } from '@angular/core';
+import{ Router } from '@angular/router';
```
Comment (tsangste) [inline src/app/waitingroom/waitingroom.component.ts]: need space between import and {

File: src/system-config.ts [hunk 6]
```diff
@@ -37,7 +37,10 @@ const barrels: string[] = [
     'app/card',
     'app/card-list',
     'app/card-view',
-    /** @cli-barrel */
+    'app/client-arrival',
+  'app/client-arrival-view',
```
Comment (tsangste) [inline src/system-config.ts]: tabbing here!

**Comment by tsangste** [comment]
o btw the branching looks broken here lol as it is saying your committing commits that are already in lmao!

File: src/app/session.service.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+import { Observable } from 'rxjs';
+import { Http } from '@angular/http';
+
+@Injectable()
+export class SessionService {
+
+    constructor(private http: Http) {}
+
+    checkSession(sessionId: string): Observable<any> {
+        let address = 'http://172.16.100.181:3000';
```
Comment (quickskape) [inline src/app/session.service.ts]: Can we make this an opaque token and DI it in across the app?

**Pull request merged**